pkg/chunk/aws/s3_storage_client.go

```diff
@@ -72,6 +72,7 @@ type S3Config struct {
 	SSEEncryption    bool       `yaml:"sse_encryption"`
 	HTTPConfig       HTTPConfig `yaml:"http_config"`
 	SignatureVersion string     `yaml:"signature_version"`
+	SSEConfig        SSEConfig  `yaml:"sse_config"`
```

Comment:
In recent times we stopped appending `_config` to sub-config structs cause it's a bit redundant. I would remove it from here too.

```suggestion
	SSEConfig        SSEConfig  `yaml:"sse"`
```

Reply:
If we decide to not use `_config`, we won't be compatible with [Thanos config](https://thanos.io/tip/thanos/storage.md/#s3).

I don't mind to remove `_config`, I just thought that we wanted the config interface exactly the same as Thanos. (to not "break" any future migration)

Reply:
We're not 1:1 compatible with Thanos even in the blocks storage. It's perfectly fine. We just wanna let configure the same things, so that when we'll add KMS support to blocks storage too we can keep the same config struct.